easybuild/easyconfigs/l/libgit2/libgit2-1.7.1-GCCcore-12.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'libgit2'
+version = '1.7.1'
+
+homepage = 'https://libgit2.org/'
+description = """libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant
+linkable library with a solid API, allowing you to write native speed custom Git applications in any language
+which supports C bindings."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['17d2b292f21be3892b704dddff29327b3564f96099a1c53b00edc23160c71327']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.26.3'),
+    ('pkgconf', '1.9.5'),
+]
+dependencies = [
+    ('PCRE', '8.45'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+
+github_account = '%(name)s'
+
+sanity_check_paths = {
+    'files': ['include/git2.h', 'lib64/%(name)s.so', 'lib64/pkgconfig/%(name)s.pc'],
```

please use `SHLIB_EXT` constant:
```suggestion
    'files': ['include/git2.h', 'lib64/%%(name)s.%s' % SHLIB_EXT, 'lib64/pkgconfig/%(name)s.pc'],
```